src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person({ name, age, sex, isMarried, partnerName }) {
+  const personSex = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      <p className="Person__age">
+        {age && `I am ${age}`}
+      </p>
```

Comment:
it would be better to check age before rendering 
```suggestion
      {age && (
        <p className="Person__age">{`I am ${age}`}</p>
      )}
```